Kudu.Services/Jobs/JobsController.cs

```diff
@@ -182,7 +182,8 @@ public HttpResponseMessage InvokeTriggeredJob(string jobName, string arguments =
 
                 // Return a 200 in the ARM case, otherwise a 202 can cause it to poll on /run, which we don't support
                 // For non-ARM, stay with the 202 to reduce potential impact of change
-                return Request.CreateResponse(ArmUtils.IsArmRequest(Request) ? HttpStatusCode.OK : HttpStatusCode.Accepted);
+                return Request.CreateResponse(ArmUtils.IsArmRequest(Request) ? HttpStatusCode.OK : HttpStatusCode.Accepted,
+                    ArmUtils.AddEnvelopeOnArmRequest(_triggeredJobsManager.GetLatestJobRun(jobName), Request));
```

Comment:
That's actually how I started, and it was easy to bubble up the ID. But then I was not sure exactly what to return back to client:

- Some new minimal object that just has the ID?
- A full run object where I only populate the ID?